src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    Hello, world!
+  </body>
+</html>
```

Comment:
```suggestion
</html>

```
Add an empty line in the end of your code

Reply:
I added. But I see that at GitHub I can't see this empty line in the end. Is it normal?

Reply:
Yep. It's normal 👍